Android/app/src/androidTest/java/PageObjects/AppType.kt

```diff
@@ -0,0 +1,8 @@
+package PageObjects
```

Comment:
Could we add Salesforce copyrights everywhere?

Reply:
+1